Documentation/toolbox.md

```diff
@@ -70,12 +81,12 @@ kubectl create -f rook-tools.yaml
 
 Wait for the toolbox pod to download its container and get to the `running` state:
 ```bash
-kubectl -n rook get pod rook-tools
+kubectl -n rook get pod -l "app=rook-tools"
 ```
 
 Once the rook-tools pod is running, you can connect to it with:
 ```bash
-kubectl -n rook exec -it rook-tools bash
+kubectl -n rook exec -it $(kubectl -n rook get pod -l "app=rook-tools" -o jsonpath='{.items[0].metadata.name}') bash
```

Comment:
Is there any simpler way to exec to the tools pod? In the past I also had wanted to change this to a deployment, but then decided not to when the basic connection command became so much more complicated. We really should make this change, but would be really nice to find a simpler way to exec.

Reply:
There is a problem: if we focus to use deployment which is most appropriate from kubernetes perspective (no ordering required) the name of this pod has random suffix. Other option is to change deployment to statefulset and then we can use `root-tools-1` but I'm not sure if it is proper way. 
the sub-command (`kubectl -n rook get pod -l "app=rook-tools" -o jsonpath='{.items[0].metadata.name}'`) could be change for one of below : 
 * `kubectl -n rook get pod -o=name| grep -o -m1 "rook-tools.*"`
 * `kubectl --context k8s-test-context -n rook get pod -l app=rook-tools -o=name | sed 's/pods\///'` <- this would work only when replica is set too 1